docs/UserGuide.md

```diff
@@ -1,36 +1,40 @@
 ---
 layout: page
-title: User Guide
+title: WorkBook User Guide
 ---
 
-AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized for use via a Command Line Interface** (CLI) while still having the benefits of a Graphical User Interface (GUI). If you can type fast, AB3 can get your contact management tasks done faster than traditional GUI apps.
+WorkBook (WB) is a **desktop app for CS/tech students who are applying for internships to manage their internship applications, optimized for use via a Command Line Interface (CLI)** (CLI) while still having the benefits of a Graphical User Interface (GUI). If you can type fast, WB can get your internship management tasks done faster than traditional GUI apps.
 
-* Table of Contents
-{:toc}
```

Comment:
I think it's better to keep the {:toc}, Jekyll will probably parse for the Header tags in the .md and in that case you don't need to hard code and repeat the headers below

Reply:
toc added back in. Thanks for the heads up!